src/backend/distributed/sql/citus--12.0-1--12.1-1.sql

```diff
@@ -10,3 +10,11 @@
 #include "udfs/citus_internal_delete_placement_metadata/12.1-1.sql"
 
 #include "udfs/citus_schema_move/12.1-1.sql"
+
+
+#include "udfs/start_management_transaction/12.1-1.sql"
+#include "udfs/execute_command_on_remote_nodes_as_user/12.1-1.sql"
+#include "udfs/mark_object_distributed/12.1-1.sql"
+#include "udfs/commit_management_command_2pc/12.1-1.sql"
+
+ALTER TABLE pg_catalog.pg_dist_transaction ADD COLUMN outer_xid xid8;
```

Comment:
IMPORTANT change request: These should not be included in this file, but in src/backend/distributed/sql/citus--12.1-1--12.2-1.sql

Citus 12.1 has already been released.